README.md

```diff
@@ -5,42 +5,42 @@
 [![npm version](https://badge.fury.io/js/%40icgc-argo%2Fuikit.svg)](https://badge.fury.io/js/%40icgc-argo%2Fuikit)
 [![TypeScript](https://img.shields.io/badge/types-%20TypeScript-blue)](https://www.typescriptlang.org/)
 
-Reusable UI components for Argo.
+Reusable UI components for ARGO projects.
 
 # Development
 
-### Local
+### Local Dev
 
-Both local development and building for publishing write to the dist folder. We use `npm link` for local development and avoid adding it to `package.json`.
-`npm run watch` will point to the consuming projects react libs in the global node folder. If you get an error about multiple versions of react, chances are something went wrong here.
+UIKit:
 
-Consuming project folder eg. `platform-ui`:
+- Install dependencies: `npm ci`
+- Start watching: `npm run watch`
 
-- run `npm link` from both `react` and `react-dom` node_modules.
+Consumer project:
 
-UIKit lib:
+- `npm i <root_path_to_uikit>`
 
-- Install dependencies: `npm i`
-- Start watching: `npm run watch`
+Notes:
+
+- make sure to run `watch` first as `dist` folder is used
+- assets will require a restart of watcher
+- typescript changes often need vscode to restart the TS server
+- avoid default exports for anything that needs to be used by 3rd party
```

Comment:
"avoid default exports for anything that needs to be used by 3rd party" interesting, what happened here?

Reply:
when building with TS I ran into issues using esmodules, didn't like the nested exports essentially.
switching the build to Babel and switching to a named export style simplifies things greatly.